src/DetailsView/components/test-status-choice-group.scss

```diff
@@ -18,8 +18,8 @@
             margin-top: 0px !important;
         }
         .ms-ChoiceField-wrapper.is-inFocus .ms-ChoiceField-field {
-            @media screen and (-ms-high-contrast: active) {
-                outline: 3px highlighttext solid !important;
+            @include ms-high-contrast-override {
```

Comment:
This is just to make it more consistent; the updated `Highlight`/`HighlightText` capitalization is to match the CSS spec